website/docs/components/inputs/pulsar.md

```diff
@@ -43,6 +43,8 @@ input:
     topics: []
     subscription_name: ""
     subscription_type: shared
+    tls:
```

Comment:
Looks like the Pulsar docs haven't been refreshed in a while. They don't get picked up by automatically by the docs generator since they're not included in `public/components/all/package.go`.